vcr/issuer/issuer.go

```diff
@@ -171,6 +181,11 @@ func (i issuer) Revoke(credentialID ssi.URI) (*credential.Revocation, error) {
 		return nil, fmt.Errorf("failed to publish revocation: %w", err)
 	}
 
+	// Store the revocation after it has been published
+	if err := i.store.StoreRevocation(*revocation); err != nil {
```

Comment:
why isn't publishing it enough?

Reply:
My understanding is that the issuer keeps track of the data (credentials and revocations) it has issued. A published revocation will end up in the verifier, but currently does not make its way to the issuer.

Reply:
If you need to add it here, it was already broken?

But: the revocation should be stored when received through the network when published, for the local node as well. See `vcr/ambassador.go:92`

Reply:
Later, when we migrate to accumulator revocation, the information about which credential is revoked and when will be lost. So, since the issuer has it's own administration about issued credentials, I think we also should administer the relevant information about revocation, separate from the information received back from the network. Also, I don't like mixing the 3 roles, since we need a fourth place to integrate and we make them dependent on each other and more difficult to maintain and refactor.